apps/ledger-live-mobile/src/screens/WalletCentricAsset/index.tsx

```diff
@@ -1,24 +1,19 @@
-import React, { useMemo, useState, useCallback, useEffect } from "react";
```

Comment:
In this file we split a big component with a lot of hooks in smaller components that have their internal logic / internal state.
This helps us to prevent updates of an entire screen when only one part of that screen should be updated.